plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -264,6 +264,13 @@ def helper_usable?(helper)
   def run_helper_inspection(helper, filter, do_extract, file_store_tmp, file_store_final, scope)
     begin
       helper.inject_helper
+      if !helper.has_compatible_version?
+        raise Machinery::Errors::UnsupportedHelperVersion.new(
+          "Error: machinery-helper is not compatible with the Machinery version" \
+          " #{Machinery::VERSION}. The versions need to match."
```

Comment:
The error message should not put out the Machinery version, but the helper version and what version was expected.